File: Assets/Scripts/DamageTrigger.js
//DamageTrigger.js: A simple, variable damage trigger which can be applied to any kind of object.  
//Change the damage amount in the Inspector. 

var damage: float = 20.0;
var playerStatus : Widget_Status;

function OnTriggerEnter(){
	print("ow!");
	playerStatus = GameObject.FindWithTag("Player").GetComponent(Widget_Status);
	playerStatus.ApplyDamage(damage);
}

@script AddComponentMenu("Environment Props/DamageTrigger")

File: Assets/Scripts/DamageTrigger.cs
using UnityEngine;
using System.Collections;

namespace GrillbrickStudios
{
	[AddComponentMenu("Environment Props/DamageTrigger")]
	public class DamageTrigger : MonoBehaviour
	{
		public float damage = 20.0f;
		public Widget_Status playerStatus;

		public void OnTriggerEnter(Collider other)
		{
			print("ow!");
			playerStatus = GameObject.FindWithTag("Player").GetComponent<Widget_Status>();
			playerStatus.ApplyDamage(damage);
		}
	}
}